docker-compose-pool-combo-sgx.yaml

```diff
@@ -0,0 +1,86 @@
+# Copyright 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+version: '3.5'
+
+volumes:
+  # Named volume for WPE to share MRENCLAVE with KME. MRENCLAVE value can also be passed
+  # as a command line parameter (using the option --wpe_mr_enclave) to KME enclave manger.
+  # Then the MRENCLAVE value stored in file "wpe_mr_enclave.txt" in shared volume will be
+  # overridden by the value passed in command line.
+  pool-1:
+  pool-2:
```

Comment:
Do we need these pool volumes here. It's already defined in docker-compose-pool-combo.yaml

Reply:
Thanks. Removed.